src/test/java/org/salesforce/steps/CreateIndividualSteps.java

```diff
@@ -0,0 +1,88 @@
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+import org.testng.asserts.SoftAssert;
+import salesforce.ui.PageTransporter;
+import salesforce.ui.entities.IndividualEntity;
+import salesforce.ui.pages.lightning.individuals.IndividualFormPage;
+import salesforce.ui.pages.lightning.individuals.IndividualListPage;
+import salesforce.ui.pages.lightning.individuals.IndividualRecordPage;
+import java.util.Map;
+
+public class CreateIndividualSteps {
+
+    private PageTransporter pageTransporter;
+    private IndividualFormPage individualFormPage;
+    private IndividualEntity individualEntity;
+
+    public CreateIndividualSteps(PageTransporter pageTransporter) {
+        this.pageTransporter = pageTransporter;
+    }
+
+    @Given("^I go to the new Individual formular$")
+    public void iGoToTheNewIndividualFormular() {
+        System.out.println("==================GO TO NEW INDIVIDUAL PAGE====================");
+        individualFormPage = pageTransporter.navigateToIndividualFormPage();
+    }
+
+    @When("^I create an Individual with the following values$")
+    public void iCreateAnIndividualWithTheFollowingValues(final Map<String, String> table) {
+        System.out.println("==================CREATE INDIVIDUAL ENTITY====================");
+        ObjectMapper objectMapper = new ObjectMapper();
+        individualEntity = objectMapper.convertValue(table, IndividualEntity.class);
+        individualFormPage.createIndividual(table.keySet(), individualEntity);
+    }
+
+    @When("^I set the lastname \"(.*?)\"$")
+    public void iSetTheEntityValues(String name) {
+        individualFormPage.setLastnameTextbox(name);
+        individualFormPage.clickOnsave();
+    }
+
+    @When("^I create an individual$")
+    public void iCreateAnIndividual(final Map<String, String> table) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        individualEntity = objectMapper.convertValue(table, IndividualEntity.class);
+        individualFormPage.createIndividual(table.keySet(), individualEntity);
+    }
+
+    @Then("^The Individual's name is displayed in the header$")
+    public void theIndividualsNameIsDisplayedInTheHeader() {
+        String resultMessage = individualFormPage.getCreatedSuccessMessage();
+        Assert.assertTrue(resultMessage.contains("was created."));
+    }
+
+    @Then("^The name displayed should contain (.*) (.*) and (.*)$")
+    public void theNameDisplayedShouldContain(final String salutation, final String firstname, final String lastname) {
+        IndividualRecordPage individualRecordPage = new IndividualRecordPage();
+        String headerNameText = individualRecordPage.getNameHeaderText();
+        String fullnameWithSalutation = String.format("%s %s %s", salutation, firstname, lastname);
+        Assert.assertEquals(headerNameText, fullnameWithSalutation);

```

Comment:
Could we use a soft asser instead of a hard assert and assertAll in the post conditions?